src/js/angular-tooltips.js

```diff
@@ -77,12 +79,7 @@
 
         thisElement.bind(showTriggers, function onMouseEnterAndMouseOver() {
 
-          if (!initialized) {
-
-            initialized = true;
-            $scope.initTooltip(side);
-          }
-
+          $scope.initTooltip(side);
```

Comment:
As far as I know, resize event refers to window resize. It's not easy to know when an element resizing. :cry: 
Also, mouse events are not enough to detective elements movements.